pagefind_ui/default/svelte/ui.svelte

```diff
@@ -198,23 +197,20 @@
         role="search"
         aria-label={translate("search_label", automatic_translations, translations)}
         action="javascript:void(0);"
-        on:submit={(e) => e.preventDefault()}
+        on:submit|preventDefault ={() => {}}
     >
         <input
             class="pagefind-ui__search-input"
             on:focus={init}
-            on:keydown={(e) => {
+            on:keydown|preventDefault ={(e) => {
```

Comment:
This would be my only issue, I'd keep it as before, as you are now preventing default for *all* `keydown` events, which wasn't the case before.

```suggestion
            on:keydown={(e) => {
```

Reply:
> This would be my only issue, I'd keep it as before, as you are now preventing default for _all_ `keydown` events, which wasn't the case before.

Done :)